examples/hello-world/build.gradle.kts

```diff
@@ -0,0 +1,30 @@
+plugins {
+  `java-library`
+  `maven-publish`
+  val sigstoreVersion = System.getProperty("sigstore.version") ?: "0.11.0"
+  id("dev.sigstore.sign") version "$sigstoreVersion"
+}
+
+// required to resolve sigstore-java
+repositories {
+  mavenLocal() // for testing against dev builds
+  mavenCentral()
+}
+
+publishing {
+    publications {
+        create<MavenPublication>("maven") {
+            groupId = "com.example"
+            artifactId = "hello-world"
+            version = "1.0.0"
```

Comment:
Should this rather be at the top level?
I guess users should configure group, artifact, version anyway, so customizing the same at the publication level does not sound right.

Reply:
oh, yeah I guess this was kind of lazy on my part. Lemme update this.